src/app/Agent/AgentLiveProbes.tsx

```diff
@@ -210,7 +210,7 @@ export const AgentLiveProbes: React.FC<AgentLiveProbesProps> = (_) => {
         context.target.target(),
         context.notificationChannel.messages(NotificationCategory.ProbeTemplateApplied),
       ]).subscribe(([currentTarget, e]) => {
-        if (currentTarget?.connectUrl != e.message.targetId) {
+        if (currentTarget?.connectUrl != e.message.targetId && currentTarget?.jvmId != e.message.jvmId) {
```

Comment:
Would we just need to check `jvmId`?

Reply:
updated to just check jvmId for every event